blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineFolderImpl.java

```diff
@@ -186,14 +186,11 @@ public BlueIcon getIcon() {
 
     @Override
     public Iterable<String> getPipelineFolderNames() {
-        return Iterables.transform(getPipelines(), new Function<BluePipeline, String>() {
-            @Override
-            public String apply(@Nullable BluePipeline input) {
-                if (input != null && input instanceof BluePipelineFolder) {
-                    return input.getName();
-                }
-                return null;
+        return Iterables.transform(getPipelines(), input -> {
```

Comment:
NPE was somewhere here? Maybe if we fix it here we should be good?

Reply:
yup ezy